src/signals/IncidentMap/components/IncidentMap/IncidentMap.tsx

```diff
@@ -54,13 +63,34 @@ export const IncidentMap = () => {
   )
 
   /* istanbul ignore next */
-  const handleIncidentSelect = useCallback((incident: Incident) => {
-    setSelectedIncident(incident)
-    setDrawerState(DrawerState.Open)
-  }, [])
+  const handleIncidentSelect = useCallback(
+    (incident: Incident) => {
+      const sanitaizedCoords = featureToCoordinates(incident.geometry)
+      // When marker is underneath the drawerOverlay, move the map slightly up
+      if (map && isMobile(mode) && sanitaizedCoords.lat < map.getCenter().lat) {
+        const currentZoom = map.getZoom()
+        const coords = {
+          lat: sanitaizedCoords.lat - 0.0003,
+          lng: sanitaizedCoords.lng,
+        }
+        const zoom =
+          currentZoom > DEFAULT_ZOOM
```

Comment:
this looks kinda funky, for me its okay, maybe wrapping in 

currentZoom < 12
             ? 12
             : currentZoom

in round brackets makes it more readable?! just drop this comment if you like:)